app_dart/lib/src/model/firestore/ci_staging.dart

```diff
@@ -293,37 +308,52 @@ enum CiStage implements Comparable<CiStage> {
   String toString() => name;
 }
 
+enum StagingConclusionResult {
+  /// Check run update recorded successfully in the respective CI stage.
+  ok,
+
+  /// The check run is not in the specified CI stage.
+  ///
+  /// Perhaps it's from a different CI stage.
+  missing,
+
+  /// The check run was found in the specified CI stage but the update failed.
+  failed,
```

Comment:
Higher up:
        log.info('$logCrumb: staging document not found for $transaction');
          result: StagingConclusionResult.failed,

So the check run wasn't found either; but there was an internal error. Do we want that visibility here?

Reply:
If the document wasn't found, how can we know whether the check run would be found in the doc or not?

Reply:
Either that or we have another conclusion.  The comment is "the check run was found" - it wasn't because we don't have the document.

Reply:
Ah, gotcha. Changed it to `internalError` and updated the docs.